velox/docs/functions/spark/datetime.rst

```diff
@@ -337,3 +325,33 @@ These functions support TIMESTAMP and DATE input types.
     part of the 53rd week of year 2004, so the result is 2004. Only supports DATE type.
 
         SELECT year_of_week('2005-01-02'); -- 2004
+
+Different Behaviors Between Simple And Joda Date Formmaters
+--------------------------------
+
+To align with Spark, Velox supports both `Simple <https://docs.oracle.com/javase/8/docs/api/java/text/SimpleDateFormat.html>`_
+and `Joda <https://www.joda.org/joda-time/>`_ date formmater to parse/format timestamp/date strings
+on the functions `from_unixtime`, `unix_timestamp`, `make_date` and `to_unix_timestamp`.
```

Comment:
on the functions -> used in functions

for functions, use references 

```
:spark:func:`from_unixtime`
```

Reply:
Thanks. Fixed